style.css

```diff
@@ -1,9 +1,10 @@
 /* Design tokens */
 :root {
-  --paper: hsla(251, 28%, 88%, 0.99);
+  --paper: hsla(31, 18%, 90%, 0.99);
   --ink: hsla(244, 16%, 17%, 0.95);
   --brand: hsla(0, 79%, 63%, 0.9);
-  --font: "Raleway", system-ui, sans-serif;
+  --font: "Roboto", system-ui, sans-serif;
+  --p-font: "Poppins", system-ui, sans-serif;
```

Comment:
Is p-font a good variable name? Will you only ever want to use this on p tags? If so, do you need a variable? You can just use the p selector.

Try to use variables to describe your purpose more clearly -- more richly describing the relationships and functions of rules instead of re-stating existing rules.

A common pairing here is to use something like:
--display-font: 
--copy-font: 

Then you can apply one variable to all the places you want headings/display type and the other to the places you want body copy type, and then when someone else reads your code they can tell what you meant.